### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'catch')

### ISSUE THREAD

**Issue by jrkienle** [digital-loukoum/tsc-esm#3]
Hello! I recently installed this package and ran into an error on first use. I followed the documentation to the best of my knowledge, and am a little stuck. Here's the full error stack:

```
file:///workspaces/monorepo-tools/node_modules/.pnpm/@digitak+tsc-esm@3.1.1/node_modules/@digitak/tsc-esm/binary/tsc-esm.js:3
build().catch(error => {

TypeError: Cannot read properties of undefined (reading 'catch')
    at file:///workspaces/monorepo-tools/node_modules/.pnpm/@digitak+tsc-esm@3.1.1/node_modules/@digitak/tsc-esm/binary/tsc-esm.js:3:8
    at ModuleJob.run (node:internal/modules/esm/module_job:185:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)
 ELIFECYCLE  Command failed with exit code 1.
 ```
 
 Here's my TSConfig

 ```
 {
  "exclude": ["node_modules"],
  "extends": "@kienleholdings/base-tsconfig",
  "include": ["./src"],
  "compilerOptions": {
    "lib": ["ESNext", "DOM"],
    "module": "es2020",
    "outDir": "lib",
    "target": "es2021",
    "types": ["node"]
  }
}
```

And here's the relevant portion of my package.json

```
{
  ...
  "type": "module",
  "scripts": {
    "build": "tsc-esm"
  },
  "devDependencies": {
    "@digitak/tsc-esm": "^3.1.1",
    "typescript": "^4.5.5"
  }
  ...
}
```

Let me know if there's any other information you need! I love this solution to the esm importing problem, and hopefully one day it'll simply be a part of TypeScript

**Comment by jrkienle** [comment]
Just realized that this might be a dupe of #2 - I'd be willing to close this as long as one of the tickets gets addressed :)

**Comment by Gin-Quin** [comment]
Indeed, I now close this thanks to @jrkienle fix (available from version 3.1.2).
